docs/developers/developing.rst

```diff
@@ -135,7 +135,7 @@ To run tests and automatically re-run them whenever any source files change, run
    gulp test-watch
 
 You can filter the tests which are run by passing ``--grep <pattern>`` as an
-argument to ``gulp test``. See the documentation for Mocha's
+argument to ``make test``. See the documentation for Mocha's
```

Comment:
This section about filtering tests no longer worsk because make doesn't pass along CLI arguments to gulp. This could be revised to use `make test FILTER=<pattern>` instead using:

```diff
--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,11 @@ dev: build/manifest.json

 .PHONY: test
 test: node_modules/.uptodate
-       yarn run test
+ifdef FILTER
+       yarn test --grep $(FILTER)
+else
+       yarn test
+endif
```

Reply:
Done